test_othermlpackages/conda-tensorflow.yaml

```diff
@@ -6,6 +6,6 @@ dependencies:
 - python==3.8
 - pip>=19.1.1
 - pytest
-- tensorflow>=2.4
 - pip:
    - scikeras
+   - tensorflow # Latest TF not on conda-forge
```

Comment:
Or alternatively just `scikeras[tensorflow]`

Reply:
Doesn't work for some reason :-(